gov.pnnl.goss.gridappsd/src/gov/pnnl/goss/gridappsd/testmanager/CompareResults.java

```diff
@@ -0,0 +1,340 @@
+/*******************************************************************************
+ * Copyright � 2017, Battelle Memorial Institute All rights reserved.
+ * Battelle Memorial Institute (hereinafter Battelle) hereby grants permission to any person or entity 
```

Comment:
In some files the copyright works and some it doesn't which is strange...change this and the quotes for AS IS below